services/user/update.go

```diff
@@ -137,7 +137,7 @@ func UpdateUser(ctx context.Context, u *user_model.User, opts *UpdateOptions) er
 		} else if !user_model.IsLastAdminUser(ctx, u) /* not the last admin */ {
 			u.IsAdmin = opts.IsAdmin.Value().FieldValue // it's safe to change it from false to true (not the last admin)
 			cols = append(cols, "is_admin")
-		} else /* IsAdmin=false but this is the last admin user */ { //nolint
+		} else /* IsAdmin=false but this is the last admin user */ { //nolint:gocritic // could flatten the condition, no idea how to flatten comments at the moment
```

Comment:
This line was written by me intentionally to make it easier to read.

Sometimes the lint suggested changes are not good for understanding.

The common examples are like:

```
if  foo {
} else if x && y {
    if !u || !v {
   }
}
```

It can be flattened to like this or something:

```
if  foo {
} else if x && y && (!u || !v) {
}
```

But when the `u, v` doesn't belong to `x, y`, it will be difficult to understand.

Reply:
```suggestion
		} else /* IsAdmin=false but this is the last admin user */ { //nolint:gocritic // make it easier to read
```